### ISSUE TITLE
Feature request: internationalization

### ISSUE THREAD

**Issue by derwolle** [Home-Assistant-Pro-Team/Smart-Laundry-System---Lavatrice#1]
Hey,

looks really great, but my italian is quite bad ;). Any chance for internationalization support with separate language files?

Thx and regards
Wolfgang

**Comment by marco-hacs** [comment]
frankly there is no translation provided but generally Google translate, with a little patience, solves these problems. we are busy making new packages and there is very little time available to make them multilingual. Maybe one tomorrow...

**Comment by derwolle** [comment]
Yes, translation would be doable, but to provide it to others the usage of language files would be great. The translatation itself is the minor problem in my opinion.

**Comment by ymurawski** [comment]
Oh yes please make it international ❤️
English for now would be OK.. Maybe later other languages 🤔

**Comment by marco-hacs** [comment]
for now we are working on making more appliances, maybe when finished we can evaluate the international language. It would be nice to get collaboration for this maybe with forke. we are open to collaborations to take full advantage of the features of ha

**Comment by evilmumi** [comment]
Hi. any news on this topic ?
